index.js

```diff
@@ -7,8 +7,31 @@ module.exports = {
   premberConfig,
 
   postprocessTree(type, tree) {
+    if (type !== 'all') {
+      return tree;
+    }
+
+    return this._prerenderTree(tree);
+  },
+
+  /**
+   * This function is *not* called by ember-cli directly, but supposed to be imported by an app to wrap the app's
+   * tree, to add the prerendered HTML files. This workaround is currently needed for Embroider-based builds that
+   * don't support the `postprocessTree('all', tree)` hook used here.
+   */
+  prerender(app, tree) {
+    let premberAddon = app.project.addons.find(({ name }) => name === 'prember');
```

Comment:
@simonihmig how about add test coverage for this one and `_prerenderTree`?

Reply:
The embroider scenarios are using the `prerender` to run the whole test suite, see https://github.com/ef4/prember/pull/64/files#diff-e835654135548f75db36136807171b4d12d3514e043f4f80a146b88cc668fba4R23. 

And `_prerenderTree` is a private method, and used both by `postprocessTree` and `prerender` (basically extracted from `postprocessTree` so it can be shared with `prerender`), so already implicitly covered by prember`s node-based acceptance tests. Or what kind of test did you have in mind?